jvb/src/main/kotlin/org/jitsi/videobridge/AudioSubscription.kt

```diff
@@ -32,9 +32,12 @@ class AudioSubscription() {
             is ReceiverAudioSubscriptionMessage.All -> return
             is ReceiverAudioSubscriptionMessage.None -> return
             is ReceiverAudioSubscriptionMessage.Include -> {
-                wantedSsrcs = sources.filter { desc ->
+                val (local, remote) = sources.partition { desc ->
```

Comment:
I'm confused. `local` are the sources from the conference, which the new subscription subscribes to? And `remote` are the sources that it doesn't subscribe to?